From 6a41300deb2611ed69c49507746bc96a634f800c Mon Sep 17 00:00:00 2001
From: Andreas Gruenbacher <agruenba@redhat.com>
Date: Thu, 9 Mar 2023 14:30:05 +0100
Subject: [PATCH 16/24] dlm_controld: get rid of unnecessary memset

In process_plocks(), we're reading in the entire info object, so there
is no need to zero out the buffer first.
---
 dlm_controld/plock.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/dlm_controld/plock.c b/dlm_controld/plock.c
index f03f3abea983..85d6fdf2819b 100644
--- a/dlm_controld/plock.c
+++ b/dlm_controld/plock.c
@@ -1523,8 +1523,6 @@ void process_plocks(int ci)
 
 	gettimeofday(&now, NULL);
 
-	memset(&info, 0, sizeof(info));
-
 	rv = do_read(plock_device_fd, &info, sizeof(info));
 	if (rv < 0) {
 		log_debug("process_plocks: read error %d fd %d\n",
-- 
2.35.3

